src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsnonassociationsapi/resource/SyncResource.kt

```diff
@@ -86,11 +92,19 @@ class SyncResource(
       ),
     ],
   )
-  fun sync(
+  fun delete(
     @Valid @RequestBody
-    syncRequest: DeleteSyncRequest,
+    deleteRequest: DeleteSyncRequest,
   ) =
-    syncAndMigrateService.delete(syncRequest)
+    syncAndMigrateService.delete(deleteRequest)
+      .onEach { deletedNonAssociation ->
+        eventPublishService.publishEvent(
+          NonAssociationDomainEventType.NON_ASSOCIATION_DELETED,
+          deletedNonAssociation,
+          deleteRequest,
+          "nomis",
```

Comment:
I could go with this , but in other syncs / migrations we have used values 'NOMIS' and 'DPS'